pysatModels/utils/match.py

```diff
@@ -208,7 +211,12 @@ def collect_inst_model_pairs(start, stop, tinc, inst, inst_download_kwargs={},
     istart = start
     while start < stop:
         # Load the model data for each time
-        mdata = model_load_rout(start, **model_load_kwargs)
+        try:
+            mdata = model_load_rout(start, **model_load_kwargs)
+        except (IOError, ValueError) as err:
+            pysatModels.logger.info(
+                'unable to load model data at {:}\n{:}'.format(start, err))
+            mdata = None
```

Comment:
Is the intent of this to skip over times that don't exist in the model?

Reply:
Yes, or files that are corrupted but are still present because you don't have permission to alter the achieve.

Reply:
The unit tests currently expect a ValueError here when there is no data.  Should we adjust the tests to look for a `None`?

Reply:
I did that initially and found it wasn't as informative.  There are many reasons there won't be data and this way you know why.